src/Utility/FieldFunctions.C

```diff
@@ -286,6 +286,16 @@ FieldFunction::FieldFunction (const std::string& fileName,
 }
 
 
+FieldFunction::FieldFunction (const std::vector<Field*>& fields)
+  : FieldFuncScalarBase("", "", "",  -1)
```

Comment:
Maybe equip `FieldFuncScalarBase`with a default constructor such that this explicit forwarding is not needed?

Reply:
sure, i also had to do it to FieldFuncHDF5.